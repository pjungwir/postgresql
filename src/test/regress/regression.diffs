*** /Users/paul/src/postgres/src/test/regress/expected/without_overlaps.out	2018-11-20 20:02:17.000000000 -0800
--- /Users/paul/src/postgres/src/test/regress/results/without_overlaps.out	2018-11-20 20:02:20.000000000 -0800
***************
*** 16,46 ****
--- 16,74 ----
    CONSTRAINT without_overlaps_pk PRIMARY KEY (id, WITHOUT OVERLAPS valid_at)
  );
  -- PK with a PERIOD that isn't there:
+ -- TODO
  -- PK with a non-range column:
  CREATE TABLE without_overlaps_test (
    id INTEGER,
    valid_at TEXT,
    CONSTRAINT without_overlaps_pk PRIMARY KEY (id, WITHOUT OVERLAPS valid_at)
  );
+ ERROR:  relation "without_overlaps_test" already exists
  -- PK with one column plus a range:
  CREATE TABLE without_overlaps_test (
    id INTEGER,
    valid_at tstzrange,
    CONSTRAINT without_overlaps_pk PRIMARY KEY (id, WITHOUT OVERLAPS valid_at)
  );
+ ERROR:  relation "without_overlaps_test" already exists
  -- PK with two columns plus a range:
+ -- TODO
  -- PK with one column plus a PERIOD:
+ -- TODO
  -- PK with two columns plus a PERIOD:
+ -- TODO
  -- PK with a custom range type:
+ -- TODO
  --
  -- test PK inserts
  --
  -- okay:
  INSERT INTO without_overlaps_test VALUES (1, tstzrange('2018-01-02', '2018-02-03'));
+ ERROR:  INSERT has more expressions than target columns
+ LINE 1: INSERT INTO without_overlaps_test VALUES (1, tstzrange('2018...
+                                                      ^
  INSERT INTO without_overlaps_test VALUES (1, tstzrange('2018-03-03', '2018-04-04'));
+ ERROR:  INSERT has more expressions than target columns
+ LINE 1: INSERT INTO without_overlaps_test VALUES (1, tstzrange('2018...
+                                                      ^
  INSERT INTO without_overlaps_test VALUES (2, tstzrange('2018-01-01', '2018-01-05'));
+ ERROR:  INSERT has more expressions than target columns
+ LINE 1: INSERT INTO without_overlaps_test VALUES (2, tstzrange('2018...
+                                                      ^
  INSERT INTO without_overlaps_test VALUES (3, tstzrange('2018-01-01', NULL));
+ ERROR:  INSERT has more expressions than target columns
+ LINE 1: INSERT INTO without_overlaps_test VALUES (3, tstzrange('2018...
+                                                      ^
  -- should fail:
  INSERT INTO without_overlaps_test VALUES (1, tstzrange('2018-01-01', '2018-01-05'));
+ ERROR:  INSERT has more expressions than target columns
+ LINE 1: INSERT INTO without_overlaps_test VALUES (1, tstzrange('2018...
+                                                      ^
  INSERT INTO without_overlaps_test VALUES (NULL, tstzrange('2018-01-01', '2018-01-05'));
+ ERROR:  INSERT has more expressions than target columns
+ LINE 1: INSERT INTO without_overlaps_test VALUES (NULL, tstzrange('2...
+                                                         ^
  INSERT INTO without_overlaps_test VALUES (3, NULL);
+ ERROR:  INSERT has more expressions than target columns
+ LINE 1: INSERT INTO without_overlaps_test VALUES (3, NULL);
+                                                      ^

======================================================================

