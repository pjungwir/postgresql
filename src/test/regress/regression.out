test without_overlaps             ... FAILED
